storage/src/main/java/org/apache/kafka/storage/internals/log/LogConfig.java

```diff
@@ -179,6 +182,7 @@ public Optional<String> serverConfigName(String configName) {
     public static final boolean DEFAULT_REMOTE_STORAGE_ENABLE = false;
     public static final long DEFAULT_LOCAL_RETENTION_BYTES = -2; // It indicates the value to be derived from RetentionBytes
     public static final long DEFAULT_LOCAL_RETENTION_MS = -2; // It indicates the value to be derived from RetentionMs
+    public static final String DEFAULT_REMOTE_LOG_DISABLE_POLICY = "retain";
```

Comment:
Can you declare the respective fields for valid values like "retain" and "delete"?

Reply:
Sorry, I am not certain I understand - the two options with respect to validation are already defined on line 283 i.e. `in("retain", "delete")`. Do you want something along the lines of:
```
public static final List<String> REMOTE_LOG_DISABLE_POLICY_VALID_VALUES = Arrays.asList("retain", "delete");

...

.defineInternal(TopicConfig.REMOTE_LOG_DISABLE_POLICY_CONFIG, STRING, DEFAULT_REMOTE_LOG_DISABLE_POLICY, in(REMOTE_LOG_DISABLE_POLICY_VALID_VALUES),
                MEDIUM, TopicConfig.REMOTE_LOG_DISABLE_POLICY_DOC);
```

Reply:
I think satish's comment is to declare constants for `retain` and `delete`

Reply:
nit: 

Please use the defined constant: "retain" -> TopicConfig.REMOTE_LOG_DISABLE_POLICY_RETAIN